src/setup.py

```diff
@@ -39,7 +39,8 @@ def initialize_options(self):
     def finalize_options(self):
         pass
     def run(self):
-        os.system('rm -vrf ./build ./dist ./*.pyc ./*.egg-info | find -iname "*.pyc" -exec rm {} +')
+        os.system('rm -vrf ./build ./dist ./*.pyc ./*.egg-info')
+        os.system('find -iname "*.pyc" -exec rm {} +')
```

Comment:
we think that there is a problem in this command.
If the user is working in / for example, it will delete all pyc files on the entire file system.
Please test it ;)
Proposition: Use basename function to determine the working directory.